BMM.Core/Implementations/PlayObserver/PlayStatistics.cs

```diff
@@ -243,6 +243,10 @@ public async Task WriteEvent(TrackPlayedEvent ev)
                 {"sentAfterStartup", ev.SentAfterStartup},
                 {"playbackOrigin", ev.PlaybackOrigin}
             };
+            if (_config.UseAnalyticsId)
+                dict.Add(nameof(User.AnalyticsId), ev.AnalyticsId);
+            else
+                dict.Add(nameof(User.PersonId), ev.PersonId);
```

Comment:
Add one line space above and below this if..else statement.

Reply:
done